test-utils/src/index.js

```diff
@@ -31,3 +31,35 @@ export function act(cb) {
 	options.debounceRendering = Component.__test__previousDebounce;
```

Comment:
Just a random thought, not something to necessarily address in this PR: How do `setupRerender()` and `act()` interact? If I have a test that calls `setupRerender` in a `beforeEach` and then a test that uses `act`, do they behave well together?

Reply:
Yes, I don't think this still belongs here.